uptrain/tests/test_monitors_concept_drift_adwin.py

```diff
@@ -90,8 +90,7 @@ def plot_data(drifts=None):
                 "type": uptrain.Monitor.CONCEPT_DRIFT,
                 "algorithm": uptrain.DataDriftAlgo.ADWIN,
                 "measurable_args": {
-                    "type": uptrain.MeasurableType.MAE,
-                    "feature_name": "data",
```

Comment:
@a-r-r-o-w I don't understand this, why is feature_name no longer needed?

Reply:
@vipgupta I think, if I recall correctly, my reasoning was that only outputs and ground truths are required in computing the concept drift and the feature_name doesn't end up being used anywhere in the concept drift calculation part.

Reply:
Got it